### ISSUE TITLE
Continent - curation before uploading first vocabulary version

### ISSUE THREAD

**Issue by ManonGros** [gbif/vocabulary#72]
Here is a file to edit: https://drive.google.com/file/d/1BMSJ5-hd0b8MXx-p-rQe9CIbQEa9dZHC/view?usp=sharing

**It contains:**

- the list of existing concepts
- a list of the values already mapped to the concepts (they are all in the **Hidden** sheet/tab for now)
- the GBIF verbatim values for this field that appear more than 10,000 times or in 5 or more datasets

Pease check instructions here: https://github.com/gbif/vocabulary/issues/70

**Comment by ManonGros** [comment]
I think I have finished this vocabulary. @timrobertson100 and perhaps @MattBlissett, could you take a look and let me know if this seems ok to you?

**Comment by MattBlissett** [comment]
This looks like a vocabulary for GBIF Region (a classification for GBIF's use in organizing activities) rather than continent (seven-continent geographical definition, including several countries like RU, KZ, TR with land on two continents).

The concept label_en for EUROPE and SOUTH AMERICA should be Europe and South America.

Central America and the Caribbean are part of the South American continent.

**Comment by tucotuco** [comment]
@Matthew Blissett ***@***.***> Are you sure you don't mean that
Central America and the Caribbean are part of North America? I have never
seen the grouping you suggest.

**Comment by MattBlissett** [comment]
Yes, I mean Central America and the Caribbean are part of North America.

The vocabulary spreadsheet as it is at the moment has them as part of "South America / Latin America and the Caribbean".

**Comment by ManonGros** [comment]
Thanks for checking! you are right, I used the GBIF regions because I didn't think.
I will correct that.

**Comment by ManonGros** [comment]
Ok I updated the names and the mapping. Could I ask you to check again? Thanks!

**Comment by MattBlissett** [comment]
`NA` could be North America, but could also be "Not applicable" (or Namibia), is that why you excluded that one?

It seems fine now, thanks.

**Comment by ManonGros** [comment]
yes it is why I excluded it. I thought it was too ambiguous.

**Comment by tucotuco** [comment]
Good choice.

**Comment by MattBlissett** [comment]
I've changed:

- Label_en of Antarctica was misspelt.
- Removed South & Central America, Central America is part of the North American continent
- Removed the oceans and seas which aren't part of the continents.

It's now ready to import.

**Comment by CecSve** [comment]
Thank you! I will import it to the server.
